README-ZH_CN.md

```
@@ -29,82 +58,42 @@ Windows PowerShell：
-- OpenClaw：`${OPENCLAW_HOME:-~/.openclaw}/skills/oo` 和
-  `${OPENCLAW_HOME:-~/.openclaw}/skills/oo-find-skills`
-- QoderWork：`~/.qoderwork/skills/oo` 和
-  `~/.qoderwork/skills/oo-find-skills`
-
-之后你就可以在任一受支持宿主中使用它们。比如在 Codex 中：
-
-```text
-$oo 帮我生成 OOMOL 字符串的二维码
-```
+> /oo 总结今天未读的 Gmail 邮件。
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)